### PR TITLE
Add notes on docker image architecture to LibreELEC/documentation/docker.md

### PR DESCRIPTION
**Pull request by aannenko**

### CODE REVIEW

File: installation/docker.md [hunk 1]
```diff
@@ -4,3 +4,11 @@ The awesome folks at [LinuxServer.io](https://linuxserver.io) maintain a range o
 
 It is also possible to install LinuxServer Docker containers from the SSH console, which is also how to access the full [fleet of LinuxServer containers](https://fleet.linuxserver.io).
 
+## A note on Docker image architectures
```
Comment (chewitt) [inline installation/docker.md:7]: Container Architecture

Comment (aannenko) [reply installation/docker.md:7]: I'd say that the images really dictate where containers can be spun up from them, but ok NP :)

File: installation/docker.md [hunk 2]
```diff
@@ -4,3 +4,11 @@ The awesome folks at [LinuxServer.io](https://linuxserver.io) maintain a range o
 
 It is also possible to install LinuxServer Docker containers from the SSH console, which is also how to access the full [fleet of LinuxServer containers](https://fleet.linuxserver.io).
 
+## A note on Docker Container Architecture
+
+LibreELEC's kernel and user space may support different architectures. For instance, starting with version 10, LibreELEC on arm64-capable devices (e.g. Raspberry Pi 3/4) uses a 64-bit kernel whereas the user space is 32-bit.
+
+For Docker this means that it can only run containers from the images compatible with LibreELEC's user space. For example, if you want to run a Docker container in LibreELEC 11 on Raspberry Pi 4, the Docker image should be built for the platform `linux/arm` (same as `linux/arm/v7` or `linux/arm/v8`), but not `linux/arm64` (same as `linux/arm64/v8`) as Docker will fail to run those.
+
+Starting with LibreELEC 12, certain releases of LibreELEC may have 64-bit user space to accompany a 64-bit kernel.
```
Comment (heitbaum) [inline installation/docker.md:13]: Suggested reword “Starting with LibreELEC 12, certain ARMv8 releases of LibreELEC have a 64-bit user space to accompany the 64-bit kernel.”

Comment (aannenko) [reply installation/docker.md:13]: What about Generic x86_64? Will it remain 32-bit or is it already x64?

Comment (heitbaum) [reply installation/docker.md:13]: > What about Generic x86_64? Will it remain 32-bit or is it already x64?

Already 64

**Pull request merged**